ocs_ci/ocs/ocp.py

```diff
@@ -358,15 +359,21 @@ def get_resource_status(self, resource_name):
             str: The status returned by 'oc get' command not in the 'yaml'
                 format
         """
-        status_index = None
+        # Get the resource in str format
         resource = self.get(resource_name=resource_name, out_yaml_format=False)
+        # get the list of titles
+        titles = re.sub('\s{2,}', ',', resource)
+        titles = titles.split(',')
+        # Get the index of 'STATUS'
+        status_index = titles.index('STATUS')
         resource = shlex.split(resource)
-        for idx, i in enumerate(resource):
-            if i.isupper() and i == 'STATUS':
-                status_index = idx
-            if not i.isupper():
-                break
-        resource_info = [i for i in resource if not i.isupper()]
+        # Get the values from the output including access modes in capital
+        # letters
+        resource_info = [
+            i for i in resource if (
+                not i.isupper() or i in ('RWO', 'RWX', 'RXO')
```

Comment:
`RXO` should be changed to `ROX`

Reply:
https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes

Reply:
fixing - thanks!